src/main/resources/application.yml

```diff
@@ -46,3 +46,9 @@ spring:
 mapping:
   data:
     code-system-entry-url: ${CODE_SYSTEM_ENTRY_URL:https://madie-dev-static.s3.amazonaws.com/mappings/code-system-entry.json}
+
+logging:
+  level:
+    root: INFO
+    gov.cms.madie.terminology: DEBUG
```

Comment:
We might want to put this one to info at least and just put it to debug as needed for special occasions